skorch/utils.py

```diff
@@ -466,3 +467,19 @@ def get_map_location(target_device, fallback_device='cpu'):
             ), DeviceWarning)
         map_location = torch.device(fallback_device)
     return map_location
+
+
+class TeeGenerator:
+    """Stores a function that produces a generator and calls list on the
```

Comment:
The docstring is no longer up-to-date. Also, could you add 2 newlines after the first sentence?